docs/serving/samples/hello-world/helloworld-shell/index.md

```diff
@@ -5,4 +5,206 @@ weight: 1
 type: "docs"
 ---
 
-{{% readfile file="README.md" %}}
+This guide describes the steps required to create the `helloworld-go` sample app and deploy it to your
```

Comment:
Shouldn't you name this something else, for instance `helloworld-shell` since the current name is confusing with the Golang sample?

Reply:
good catch that's just a c&p gotcha